scripts/jenkins-yoctobuild-build.sh

```diff
@@ -1088,15 +1101,31 @@ run_backend_integration_tests() {
 
         local testing_status=0
 
+
         if [ -f $WORKSPACE/integration/docker-compose.enterprise.yml ]; then
             cd $WORKSPACE/integration/backend-tests && \
-                PYTEST_ARGS="-k 'not Enterprise and not Multitenant'" ./run && \
-                PYTEST_ARGS="-k Enterprise" ./run -f=../docker-compose.enterprise.yml -f=../docker-compose.storage.minio.yml || \
-                testing_status=$?
+
+                case $INTEGRATION_TEST_SUITE in
+                    "enterprise")
+                        PYTEST_ARGS="-k Enterprise" ./run -f=../docker-compose.enterprise.yml -f=../docker-compose.storage.minio.yml || \
+                        testing_status=$?
+                        ;;
+                    "open")
+                        PYTEST_ARGS="-k 'not Enterprise and not Multitenant'" ./run || \
+                        testing_status=$?
+                        ;;
+                    *)
+                        PYTEST_ARGS="-k 'not Enterprise and not Multitenant'" ./run && \
+                        PYTEST_ARGS="-k Enterprise" ./run -f=../docker-compose.enterprise.yml -f=../docker-compose.storage.minio.yml || \
```

Comment:
Is there a difference between running pytest twice with to complementary filters than running it once with no filters?

Reply:
one set runs with default compose files, and the other one - with custom files.

Reply:
fixed

Reply:
in other words there would be no difference but each run uses a different setup

Reply:
Thanks for clarification. It was right there, but I couldn't see the detail of the extra files :P